### DISCUSSION TITLE
woo checkout

### DISCUSSION THREAD

**Discussion by kelderkolddk** [woocommerce/woocommerce#51758] [Checkout Flow]
The phone field, I would like to change the name from phone to mobile phone, today we only need mobile phone number, so transport companies etc. can send sms notifications.

**Comment by alexflorisca** [comment]
Thanks @kelderkolddk, I hear you. I think most people do. I think most people are also used to putting down their mobile phone number in a phone field? The easiest way we could do this is to add a filter so that it can be changed programmatically, but I'm not sure how impactful that is yet

**Comment by nielslange** [comment]
Hello @kelderkolddk 👋

Thank you for your question! When customising the phone label in the WooCommerce Checkout block, you can use the `woocommerce_get_country_locale` filter to adjust the labels based on the country. Here are two code snippets you can use:

```php
// Changes the phone label to 'Mobile' only for the Netherlands
add_filter(
    'woocommerce_get_country_locale',
    function ( $locale ) {
        $locale['NL']['phone']['label'] = __( 'Mobile', 'woocommerce' ); // Retrieves the translated string for 'Mobile'
        return $locale;
    },
    10,
    1
);
```

```php
// Changes the phone label to 'Mobile' for all countries
add_filter(
    'woocommerce_get_country_locale',
    function ( $locales ) {
        foreach ($locales as $country_code => &$locale) {
            $locale['phone'] = array('label' => __( 'Mobile', 'woocommerce' )); // Applies the translation to all country locales
        }
        return $locales;
    }
);
```

You can add either of these snippets to your website using the [Code Snippets](https://wordpress.org/plugins/code-snippets/) plugin or by adding them directly to your theme's `functions.php` file. This modification will help ensure that the phone field label meets your needs, regardless of the country selected during checkout.

Please note that `__( 'Mobile', 'woocommerce' )` retrieves the corresponding translation for 'Mobile'. For instance, if your site's language is set to Dutch, the label will display as 'Mobiel'. You can view and contribute to translations on the [WooCommerce Dutch translation page](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/nl/default/?filters%5Bstatus%5D=either&filters%5Boriginal_id%5D=16909958&filters%5Btranslation_id%5D=111781530).
